godot-core/src/registry.rs

```diff
@@ -241,6 +262,21 @@ fn fill_into<T>(dst: &mut Option<T>, src: Option<T>) {
     }
 }
 
+#[cfg(since_api = "4.1")]
+fn add_editor_plugin(class_name: &ClassName) {
+    unsafe { interface_fn!(editor_add_plugin)(class_name.string_sys()) };
+}
+
+#[cfg(before_api = "4.1")]
+fn add_editor_plugin(class_name: &ClassName) {
+    log::godot_error!(
+        "\
+        Could not register {class_name} as an editor plugin, \
+        as editor plugins are only supported in Godot 4.1+\
+    "
```

Comment:
Could the two quotes be aligned? Or opening quote on the line of `log::godot_error!(`?